dashboard/app/models/school.rb

```diff
@@ -60,14 +60,15 @@ def census_for_year(year)
   # Determines if school meets Amazon Future Engineer criteria.
   # Eligible if the school is any of the following:
   # a) title I school,
-  # b) >40% URM students,
-  # or c) >40% of students eligible for free and reduced meals.
+  # b) rural school,
+  # c) >40% URM students,
```

Comment:
```suggestion
  # c) >30% URM students,
```
per eligibility requirements here https://code.org/afe

Reply:
Oh good thing we're taking a look here and udpating it! On it!